### PR TITLE
Allow requesting just one product

### LINKED ISSUE #53: Implement the product detail view
As of now, the list view already gives all the details, including the unnecessary ones. Refactor that view and add the more comprehensive detail view.

### PR DESCRIPTION
**Pull request by illright**
Closes #53

### CODE REVIEW

File: innopoints/views/product.py [hunk 1]
```diff
@@ -134,6 +135,14 @@ def create_product():
 
 class ProductDetailAPI(MethodView):
     """REST views for the Product model"""
+    def get(self, product_id):
+        """Get a single product."""
+        product = Product.query.get_or_404(product_id)
+        schema = ProductSchema(exclude=('description',
```
Comment (aabounegm) [inline innopoints/views/product.py]: why excluding the description?

Comment (illright) [reply innopoints/views/product.py]: Right, I thought that the listing already gave all the info so I just copy-pasted. Will fix

**Review by aabounegm — APPROVED** [review]

**Comment by illright** [comment]
lol, 69

**Comment by aabounegm** [comment]
@illright how mature)

**Pull request merged**